src/modules/authorizer/role/AUT_Roles_v1.sol

```diff
@@ -131,13 +131,10 @@ contract AUT_Roles_v1 is
         _setRoleAdmin(DEFAULT_ADMIN_ROLE, ORCHESTRATOR_OWNER_ROLE);
 
         // Set up MANAGER role structure:
```

Comment:
`_setRoleAdmin` doesn't have a specific line documenting its behavior anymore, as the like `Set up MANAGER... ` is the overall heading of what's happening in all of these lines. Was this intentional? I'd have left that line in.

Reply:
Followed Omega's advice. However see your point in this